### DISCUSSION TITLE
Add "status" field to foxglove.LocationFix

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#726] [General]
**Description**

Foxglove Studio has the ability to display GPS messages in the Map panel using different colors based on the status enum of ROS `NavSatFix` messages (see https://github.com/foxglove/studio/blob/main/packages/studio-base/src/panels/Map/MapPanel.tsx#L591). There is no equivalent enum or field for `foxglove.LocationFix`, requiring the use of the ROS message to display GPS messages without a location fix in a different color.

- Version: `main`
- Platform: all

**Comment by foxhubber** [comment]
Internal tracking ticket: [FG-2860](https://linear.app/foxglove/issue/FG-2860/foxglovelocationfix-missing-status-field-[foxgloveschemas-103])

**Comment by defunctzombie** [comment]
I've changed this to a feature request since this is not a bug with any current functionality. Right now the `LocationFix` message does not have a `status` field in its design.
